### ISSUE TITLE
nfcInit() failed

### ISSUE THREAD

**Issue by maklem** [devkitPro/3ds-examples#19]
I want to use the NFC-Reader to read out various nfc-chips i have lying around (Amiibos, student identity cards, ...) But even the example does not work properly :/

Compiling in "3ds-examples/nfc/" works but when run it tells:
```
libctru nfc example.
nfcInit() failed: 0xd0a17480.
Press START to exit
```

System Information: N3DS 11.4.0-37E, Luma 8.0
devkitPro: 1.6.0
devkitARM: 46

**Comment by fincs** [comment]
```
<fincs> !err 0xd0a17480
<plutobot> Module: 93 | Description: 128 | Summary: Invalid state (5) | Level: Temporary (26)
```

Pinging @yellows8

**Comment by maklem** [comment]
I've tried two things now (successfully)

1. wait a bit before trying to use the nfc-service
2. retry to initialize the nfc-service until it works

So everything is working if you give it some time to start.

**Comment by fincs** [comment]
Fixed in smealum/ctrulib@0475b59e122fbaaf410be46e844c7fd84861ea22
